semgrep/tests/e2e/snapshots/test_cli_test/test_cli_test_time/results.json

```diff
@@ -0,0 +1,17 @@
+targets/cli_test/basic/basic.py
```

Comment:
Are these supposed to be a different color?

Reply:
Not sure why they're highlighted like this. They look fine when I open in VSCode.